src/pages/Home/Map.js

```diff
@@ -3,6 +3,11 @@ import MapGL, { Marker } from 'react-map-gl';
 import AddFriendForm from './AddFriendForm/AddFriendForm';
 import InvitationSent from './AddFriendForm/InvitationSent';
 import './Home.css';
+import { GeoJsonLayer } from 'deck.gl';
+import Geocoder from 'react-map-gl-geocoder';
+import './mapbox-gl-geocoder.css';
+import FriendSearch from '../../components/Search'
```

Comment:
This is kinda confusing. I suggest using the same class name to file. So FriendSearch class should be in FriendSearch.js file.

Reply:
Changed to Friendsearch.